DEPS

```diff
@@ -3,6 +3,7 @@ use_relative_paths = True
 deps = {
   "vendor/ad-block": "https://github.com/brave/ad-block.git@aa0485f374365a1cc3b85de85e952d6599c4437f",
   "vendor/autoplay-whitelist": "https://github.com/brave/autoplay-whitelist.git@458053a3c95b403cbe0872f289a2aafa106ee9d8",
+  "vendor/extension-whitelist": "https://github.com/brave/extension-whitelist.git@de983143e1a5052f16e8233e34a618ce67fe1bd4",
```

Comment:
please set up branch protection: https://github.com/brave/internal/issues/150

and make sure the users who have write access to this repo are as minimal as possible

Reply:
Pretty sure @bbondy needs to do this as I don't have admin rights (yay!)

Reply:
I set for only @pilgrim-brave to have access and adjusted to the default branch protection permissions.

Reply:
thanks!

Reply:
Can we remove the vetted **test** extension IDs now? Do tests all still pass?
I think they were there because we used to only allow install for vetted extensions, now we only use that for messaging. 

If tests still pass, I think you can remove them from this file:
https://github.com/brave/extension-whitelist/blob/master/data/whitelist.json

Reply:
Please add a note to the https://github.com/brave/extension-whitelist repo README to explain what a vetted extension means. Currently it only means different messaging when you install.  I just don't want someone to wrongly misinterpret what it means, social media has a way of taking things out of context sometimes and have a mob mentality about it.

Reply:
nit: Standard style is usually used for JS files.
https://github.com/brave/extension-whitelist/blob/master/scripts/gen_data_file.js#L6
So mainly change `"` to `'` and remove semicolons. 
You can change all fo the `var` to `const` except fo the ones inside the `for`.

Reply:
https://github.com/brave/extension-whitelist/blob/master/extension_whitelist_parser.cc

These would need to come before the `strcpy`
```
  if (nullptr == extensionData.sExtensionID)
    return;
```

Reply:
I think you can remove `mhjfbmdgcfjbbpaeojofohoefgiehjai` from the JSON file because I think it would be covered below.